topostats/plottingfuncs.py

```diff
@@ -38,7 +38,7 @@ def __init__(
         image_type: str = "non-binary",
         image_set: str = "core",
         core_set: bool = False,
-        interpolation: str = "nearest",
+        pixel_interpolation: str = "nearest",
```

Comment:
If the aim is to have the default as `None` then whilst the value carries through from the configuration when called from `run_topostats` it would probably be wise to change the value here to be `None`.